python/ray/dashboard/client/package.json

```diff
@@ -31,6 +31,7 @@
     "usehooks-ts": "^2.9.4"
   },
   "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```

Comment:
Do you need this dependency? If so, it might be helpful to add a comment on why in the PR description!

Reply:
Hi Tej, I’m not entirely sure where it came from—it might have been accidentally added during some updates or tool installations. I tested without it, and everything works just fine. I’ve gone ahead and removed that line in my next commit! Let me know if you notice anything.